jenkins/ci.suse.de/cloud-crowbar7.yaml

```diff
@@ -28,3 +28,14 @@
     jobs:
         - '{crowbar_job}'
 
+- project:
+    name: cloud-crowbar7-job-image-update
+    cloud_image_update_job: '{name}'
+    triggers:
+    openstack_cloud_job: cloud-crowbar7-job-x86_64
+    download_image_url: http://download.suse.de/ibs/Devel:/Cloud:/Images/images-SLE_12_SP2/ardana-jeos-SLE12SP2.x86_64.qcow2.xz
```

Comment:
It would be nice to remove the `ardana` reference in the image name as well if the image is not specific to Ardana.

Reply:
Thanks @nicolasbock , and yes, you're absolutely right. I have that on my todo list for https://jira.suse.de/browse/SCRD-8475 , but that will require some more IBS refactoring work.

Reply:
@nicolasbock on that same note, the ECP image name is wrong, as this isn't exactly a cleanvm image :)

Reply:
Ah, which image is that? I can rename it :smile: